<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskyDesignClass</class>
 <widget class="QMainWindow" name="TaskyDesignClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>494</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tasky</string>
  </property>
  <property name="windowIcon">
   <iconset resource="taskydesign.qrc">
    <normaloff>:/new/prefix1/logo.ico</normaloff>:/new/prefix1/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#TaskyDesignClass{
background-color: transparent; 
padding: 0px 0px 5px 0px;
}</string>
  </property>
  <widget class="QWidget" name="MainPart">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>580</width>
     <height>375</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>650</width>
     <height>495</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#MainPart{
	background-image: url(:/new/prefix1/Resources/background_main.png);
    backgrund-position: bottom left;
}</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>611</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="VerticalLayout">
     <item>
      <widget class="QTextEdit" name="DisplayPanel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>DisplayPanel</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit#DisplayPanel{
	background-color: #294B44;
    border-radius: 8px;
    border: 1px solid white;
    color: #FFFFFF;
    padding: 10px 10px 0px 5px;
}
QTextEdit#DisplatPanel:hover{
    background-color: black;
}
QTextEdit#DisplayPanel QScrollBar:vertical{
    background-color: #233333;
    width: 5px;
}
QTextEdit#DisplayPanel QScrollBar::handle:vertical{
   height:200px;
   background-color: #2E334C;
}
QTextEdit#DisplayPanel QScrollBar:horizontal{
    background-color: #233333;
    width: 5px;
}
QTextEdit#DisplayPanel QScrollBar::handle:horizontal{
   background-color: #2E334C;
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="HorizontalLayout">
       <property name="topMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="InputLabel">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFF00;
border: 0px solid white;</string>
         </property>
         <property name="text">
          <string>  &gt; </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="InputBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>InputBox</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit#InputBox{
color: #FFFFFF;
border: 1px solid #FFFFFF;
border-radius: 2px;
background-color: #2E534C;
}</string>
         </property>
         <property name="placeholderText">
          <string>input your command here</string>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::VisualMoveStyle</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Close</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#CloseButton{	
    border-style:flat;
	background-image: url(:/new/prefix1/Resources/button_close.png);
}
QToolButton#CloseButton:hover{
    border-style:flat;
	background-image: url(:/new/prefix1/Resources/button_close.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="taskydesign.qrc">
      <normaloff>:/new/prefix1/Resources/button_close50.png</normaloff>:/new/prefix1/Resources/button_close50.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonFollowStyle</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="MinimizeButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Hide</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#MinimizeButton{
    border-style: flat;
	background-image: url(:/new/prefix1/Resources/button_minimize50.png);
}
QToolButton#MinimizeButton:hover{
    border-style: flat;
	background-image: url(:/new/prefix1/Resources/button_minimize.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="taskydesign.qrc">
      <normaloff>:/new/prefix1/Resources/button_minimize.png</normaloff>:/new/prefix1/Resources/button_minimize.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="StatusBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>StatusBar</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Tasky">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#Tasky{
    color: yellow;
}</string>
    </property>
    <property name="text">
     <string>Tasky</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>InputBox</tabstop>
  <tabstop>DisplayPanel</tabstop>
 </tabstops>
 <resources>
  <include location="taskydesign.qrc"/>
 </resources>
 <connections/>
</ui>
